website-builder/Dockerfile

```diff
@@ -32,6 +29,7 @@ ADD Gemfile.lock Gemfile.lock
 RUN bundle install 
 
 WORKDIR $SITE_HOME
+VOLUME [ $SITE_HOME ]
```

Comment:
@uidoyen, do we truly need this? I just had removed it (see https://github.com/debezium/docker-images/commit/00a1d9b482b1f6b55edd44b0b43edbaba85b144f), as it gave me problems with recent Docker versions. And it shouldn't be part of the Dockerfile itself to begin with, but can be specified upon docker run... . WDYT?

Reply:
Actually, I have added it test locally. I am removing it again. Thanks!

Reply:
Excellent, thx!